### ISSUE TITLE
Difference between uploading vs done status

### ISSUE THREAD

**Issue by danielbyun** [fortana-co/react-dropzone-uploader#57]
What is the difference between the two statuses? I am uploading video files to S3 and it uploads super fast but will take long from 100% of uploading to receiving `headers_received` status.

I just need some clarification for UX purposes!

Thank you for this wonderful component.

**Comment by danielbyun** [comment]
I am uploading a video file to the server, and the progress bar goes pretty quickly but it's not the correct progress bar because it hasn't been sent to the server yet.

**Comment by colin-byrne-1** [comment]
@danielbyun I agree this is an issue... I am not a project owner here, but I'd love to fix this if I find the time.

**Comment by iiitmahesh** [comment]
@danielbyun have you find any solution for that???

**Comment by danielbyun** [comment]
@iiitmahesh
I just found a workaround for it - I upload it, then if it at any point fails or the user cancels (by checking the status) I call the server again and delete the file that was uploaded to the cloud.

I hope that helps, and I hope that the documentation for this component can be improved!
This is a great component and I found it very useful.

@cobyrne09 that would be awesome!
